packages/advanced-logic/test/extensions/payment-network/erc20/fee-proxy-contract.test.ts

```diff
@@ -595,4 +629,43 @@ describe('extensions/payment-network/erc20/fee-proxy-contract', () => {
       }).toThrowError('feeAmount is not a valid amount');
     });
   });
+
+  describe('applyActionToExtension/declarePayment', () => {
+    it('can applyActionToExtensions of declarePayment', () => {
```

Comment:
It is relevant to test that you can declare the payment in two halves for instance.